examples/tracer-web/examples/document-load/index.js

```diff
@@ -4,12 +4,15 @@ import { WebTracerProvider } from '@opentelemetry/web';
 import { DocumentLoad } from '@opentelemetry/plugin-document-load';
 import { ZoneContextManager } from '@opentelemetry/context-zone';
 import { CollectorTraceExporter } from '@opentelemetry/exporter-collector';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
 
-const provider = new WebTracerProvider({
-  plugins: [
-    new DocumentLoad(),
-  ],
+const provider = new WebTracerProvider();
+
+registerInstrumentations({
```

Comment:
Is there a specific reason for using `registerInstrumentations()` here instead of just calling `new DocumentLoad()` a few lines below after `provider.register()`?

Reply:
just to have consistency and show one method to load new and old, previously loading was inside WebTracerProvider, now we can use `registerInstrumentations` in both.

Reply:
I would move it down after `provider.register()` to avoid the double creation of the `Tracer` in the instrumentation.
And if people move away from `registerInstrumentations()` this is even more important.

Reply:
Basically it should not matter when you call it, it should work fine in both ways, the user should not pay attention to the order so I would not even recommended one way or another.